### ISSUE TITLE
Not working in sidePanel webextension

### ISSUE THREAD

**Issue by JVariance** [WICG/document-picture-in-picture#88]
I'm working on a webextension and tried `documentPictureInPicture.requestWindow()` from within a sidePanel. Unfortunately it doesn't work from there (onenter and window attributes are both null).

**Comment by beaufortfrancois** [comment]
@steimelchrome Is this expected that the API doesn't work from extension pages?

**Comment by steimelchrome** [comment]
Yes we don't currently allow the API to work from extension pages in Chrome. I don't know that it's technically infeasible to eventually support but since IIUC those are not attached to a WebContents then most of the code just doesn't work as is

**Comment by hanguokai** [comment]
For extension pages, my test results are:
- it works if the extension page is opened in a browser tab (like a normal web page).
- it doesn't work for extension [popup page](https://developer.chrome.com/docs/extensions/reference/action/), [side panel page](https://developer.chrome.com/docs/extensions/reference/sidePanel/) and [offscreen document](https://developer.chrome.com/docs/extensions/reference/offscreen/). The pip window closes immediately after calling `requestWindow()`
